requirements-base.txt

```diff
@@ -25,6 +25,7 @@ google-cloud-storage==1.35.0
 # Only necessary to prevent installing the latest version
 # https://github.com/googleapis/python-crc32c/issues/83
 google-crc32c==1.1.2
+isodate==0.6.1
```

Comment:
I don't know for sure but I believe there should be another team to approve dependency changes prior to merging